Tests/ApplicativeRouterHttpPipelineSupportTests/ApplicativeRouterHttpPipelineSupportTests.swift

```diff
@@ -36,7 +36,7 @@ class ApplicativeRouterHttpPipelineSupportTests: XCTestCase {
   }
 
   func testRoute_UnrecognizedWithCustomNotFound() {
-    let router = Route.home <¢ .get <*| end
+    let router = Route.iso.home <¢> get <% _end
```

Comment:
updating this test shows how similar the new router is to the old. the `iso` stuff is explained below...